docs/en/examples/elementalarea_with_dataobject.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Using ElementalArea with DataObject
+summary: Learn how to add ElementalArea to your models and manage it in a ModalAdmin
+---
+
+# Using ElementalArea with DataObject
+
+## Creating Model and adding ElementalArea
+
+In this example the class BlogPost will be created, it will extend the DataObject class. Also a new BlogPost Admin section will be created in the Admin panel where CMS users can manage the BlogPosts.
+
+Let's look at an example:
+
+**app/src/Admins/BlogPostsAdmin.php**
+
+```php
+<?php
+
+namespace App\Admins;
+
+use App\Models\BlogPost;
+use SilverStripe\Admin\ModelAdmin;
+
+class BlogPostsAdmin extends ModelAdmin
+{
+
+    private static string $url_segment = 'blog-posts-admin';
+
+    private static string $menu_title = 'Blog Posts';
+
+    private static string $menu_icon_class = 'font-icon-block-banner';
+
+    private static array $managed_models = [
+        BlogPost::class,
+    ];
+}
+```
+
+`$managed_models` - this static variable connects BlogPostAdmin with BlogPost, which will be created in the following code.
+When you create a DataObject you should define the connection between your DataObject and ElementalArea by adding `$has_one` relationship.
+In this example BlogPost will have one ElementalArea.
+
+**app/src/Models/BlogPost.php**
+
+```php
+<?php
+
+namespace App\Models;
+
+use App\Admins\BlogPostsAdmin;
+use DNADesign\Elemental\Models\ElementalArea;
+use SilverStripe\Control\Controller;
+use SilverStripe\ORM\DataObject;
+use SilverStripe\Versioned\Versioned;
+
+class BlogPost extends DataObject
+{
+    private static string $table_name = 'BlogPost';
+
+    private static $extensions = [
+        Versioned::class,
```

Comment:
Why are we explicitly saying this is `Versioned`? This should work whether the dataobject is `Versioned` or not, shouldn't it?

Reply:
I follow Steve's comment https://github.com/silverstripe/silverstripe-elemental/pull/978#pullrequestreview-955619575

Reply:
I disagree with Steve's comment :p
The block being `Versioned` should be entirely independent of the page or `DataObject` being `Versioned`.
The `DataObject` is allowed to be unversioned. The block can still have its internal draft and published states, and that's fine. Just double check that if the object (in this case `BlogPost` is saved, that publishes any unpublished blocks (which I think it will by virtue of the `$owns` config).